docs/reference/secrets-reference.md

```diff
@@ -0,0 +1,42 @@
+---
+id: secrets-reference
+title: Secrets YAML Reference
+---
+Recyclarr supports secret value substitution within it's YAML configuration files. This keeps sensitive
+values out of configuration files and makes it easier to share configurations without having to
+perform manual redaction. Using secrets is optional and support was introduced in version `TBD`.
```

Comment:
> Using secrets is optional and support was introduced in version `TBD`

You can always use the `gitversion` tool in the Recyclarr repo to see what the upcoming release is. In this case, it's going to be `v3.0.0`.

For this particular notice, please use `:::note` admonition as described in the `Version Requirement` section of the `README.md` file that I just added this morning.